readme.md

```diff
@@ -0,0 +1,39 @@
+# MariaDB continuous integration infrastructure (MDBCI)
+
+[MDBCI](https://github.com/mariadb-corporation/mdbci) is a set of tools for testing MariaDB components on the wide set of configurations. The main features of MDBCI are:
+
+* automatic creation of virtual machines according to the configuration template,
+* automatic and reliable deploy of MariaDB, Galera, MaxScale and other packages to the created virtual machines,
+* creation and management of virtual machine state snapshots,
+* reliable destruction of created virtual machines.
+
+## Read More information about MDBCI
+
+* [Requirements](docs/requirements.md)
+* [Install MDBCI](docs/quickstart.md)
+* [Create first VM](docs/simple_examples.md)
+* [All commands and commands descriptions](docs/cli_help.md)
```

Comment:
This file should die, as it duplicates the information about commands.

Instead this file must provide information on how to get help from the MDBCI itself about particular commands and their use.